src/ActionList/Item.tsx

```diff
@@ -36,8 +36,10 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
     const [slots, childrenWithoutSlots] = useSlots(props.children, {
       leadingVisual: LeadingVisual,
       trailingVisual: TrailingVisual,
-      description: Description,
+      blockDescription: [Description, props => props.variant === 'block'],
+      inlineDescription: [Description, props => props.variant !== 'block'],
```

Comment:
Something I don't understand is why not have two different slots for block and inline description? Is it to avoid breaking changes?

Reply:
Yep, that would be a breaking change. We used to support it before the regression.

But also, this feels like an implementation detail, I don't personally like the idea of it leaking out into the API with `ActionList.InlineDescription` and `ActionList.BlockDescription` 😅